app/models/account_debit.rb

```diff
@@ -0,0 +1,7 @@
+class AccountDebit < ApplicationRecord
+  belongs_to :account
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
+
+  validates :bank, presence: true
+  validates :number, presence: true
```

Comment:
Please, review if more fields should be at the presence validations too.

![image](https://user-images.githubusercontent.com/14297526/186904437-2b3527cf-1380-4a54-bc65-62ac901fe1ff.png)